src/components/Shop/CategoryBar/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import CategoryButton from "./CategoryButton";
+
+const CategoryBar: FC = () => {
+  return (
+    <Container>
```

Comment:
This works for now, but should be automatically generated when fetching the inventory.